spec/models/applicant_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+require 'date'
```

Comment:
[Ditto](https://github.com/ministryofjustice/laa-apply-for-legalaid-api/pull/5/files#r211595776)